test/runtests.jl

```diff
@@ -2,12 +2,20 @@
 using FFTW
 using Base.Test
 
-# Make super sure we don't pull in Base names
 importall FFTW
-import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
+if VERSION >= v"0.7.0-DEV.602"
+    import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
```

Comment:
I just verified that this works:
```
julia> Base.deprecate(Main.DFT, :fft)

julia> include("Foo.jl"); using .Foo

julia> fft([1,2,3])
3-element Array{Int64,1}:
 2
 3
 4
```
where `Foo.jl` is define a module `Foo` that exports `fft(x) = x+1`. 

So, it looks like we just need to add the missing `deprecate` calls to Julia 0.7 and we should be all set.